### PR TITLE
[css-align] Adds a simple TC for space-evenly & flexbox

### PR DESCRIPTION
**Pull request by frivoal**
More tests would be better, but here's a start, with a test that actually catches a bug in Chrome and Safari (but passes in Firefox).

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/5336)

### CODE REVIEW

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/218394632)

# Firefox (nightly channel) #
Testing web-platform-tests at revision 9cdff9563dbb3541c19d49affbbb7e714b7c57fe
Using browser at version BuildID 20170403100136; SourceStamp 38894655c89e68bcd8f45d31a0d3005f2c2b53db
Starting 10 test iterations
All results were stable
## All results ##
<details>
<summary>1 test ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/5336/css/css-align-3/distribution-values/space-evenly-001.html">/css/css-align-3/distribution-values/space-evenly-001.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | PASS    |          |

</details>
</details>

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/218394633)

# Chrome (unstable channel) #
Testing web-platform-tests at revision 9cdff9563dbb3541c19d49affbbb7e714b7c57fe
Using browser at version 59.0.3053.3 dev
Starting 10 test iterations
## Unstable results ##

File: css/css-align-3/distribution-values/space-evenly-001.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<!doctype html>
+<html lang=en>
+  <meta charset=utf-8>
+  <title>CSS Box Alignment: space-evenly & flexbox with single item</title>
```
Comment (mrego) [inline css/css-align-3/distribution-values/space-evenly-001.html]: I guess this is fine with `utf-8` but I'm so used to see `&amp;`. :smile:

File: css/css-align-3/distribution-values/space-evenly-001.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  <link rel="author" title="Florian Rivoal" href="https://florian.rivoal.net">
+  <meta name=flags content="">
```
Comment (mrego) [inline css/css-align-3/distribution-values/space-evenly-001.html]: Do we need this line if we don't have any flag?

File: css/css-align-3/distribution-values/space-evenly-001.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  <meta name=assert content="content-justify: space-evenly with flexbox and a single item must center it">
```
Comment (mrego) [inline css/css-align-3/distribution-values/space-evenly-001.html]: The property is called `justify-content`.

**Review by mrego — APPROVED** [review]
LGTM.

**Comment by gsnedders** [comment]
Hitting the issue in #5330 again. Can you rebase?

File: css/css-align-3/distribution-values/space-evenly-001.html (hunk 2 above)
Comment (frivoal) [inline css/css-align-3/distribution-values/space-evenly-001.html]: I don't know about the new linter, but I believe the old one wanted it, to prove that you had considered it and decided you didn't need any flag, and not just forgotten about it.

**Comment by w3c-bots** [comment]
These tests are now available on [w3c-test.org](https://w3c-test.org/submissions/5336/)

**Comment by frivoal** [comment]
> Hitting the issue in #5330 again. Can you rebase?
Done.

**Pull request merged**